bower.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "more-list",
+  "description": "An expandable list component",
+  "main": "more-list.html",
+  "dependencies": {
```

Comment:
I switched to using `yarn` and no longer need to pin any packages, because `yarn` does it :smile: